### ISSUE TITLE
The ERT documentation for the ERT workflow 'Misfit Preprocessor workflow' should be more compact and easier to read

### ISSUE THREAD

**Issue by oddvarlia** [equinor/ert#3328]
About the documentation of Misfit Preprocessor workflow:
Cna it be made more compact and easier to read. As I understand it, this is created partly automatically from source code, but is very long and not very informative. (except the first paragraph). 

**A copy of the documentation is below:**

The Misfit Preprocessor workflow provides the users with tooling to cluster and scale observations to prevent overfitting to all or parts of the observations. Examples where overfitting is likely to happen are accumulative time series and when one have data sources with severely different sampling magnitudes. This workflow is intended to be an almost out-of-the-box solution, where the user can tweak some parameters to get reasonable clusterings. The rest will be configured according to whatever is at the time considered best practices. The recommended practice for running this workflow is though the ert workflow hooks. The hook needs to run after the simulations, as simulated data is needed for the workflow. The relevant hooks are POST_SIMULATION, PRE_FIRST_UPDATE and PRE_UPDATE. The recommended hook is PRE_FIRST_UPDATE, which will only run once, also in the case where an iterative algorithm is used.

type: object

properties:

    workflow:

        must be one of:

                AutoScaleConfig

                CustomScaleConfig

        default:

            type: auto_scale

            clustering:

                type: limited_hierarchical

                linkage:

                    method: average

                    metric: euclidean

                fcluster:

                    depth: 2

            pca:

                threshold: 0.95

    observations:

        type: array

        items:

            type: string

        default:

    reports_directory:

        type: string

additionalProperties: False

definitions:

    LinkageConfig:

        The linkage implementation is backed by scipy and for a more detailed description we refer the reader to the documentation of scipy.cluster.hierarchy.linkage` (https://docs.scipy.org/doc/scipy/reference/generated /scipy.cluster.hierarchy.linkage.html).

        type: object

        properties:

            method:

                enum:

                        single

                        complete

                        average

                        weighted

                        centroid

                        ward

                type: string

                default: average

            metric:

                enum:

                        braycurtis

                        canberra

                        chebyshev

                        cityblock

                        correlation

                        cosine

                        dice

                        euclidean

                        hamming

                        jaccard

                        jensenshannon

                        kulsinski

                        mahalanobis

                        matching

                        minkowski

                        rogerstanimoto

                        russellrao

                        seuclidean

                        sokalmichener

                        sokalsneath

                        sqeuclidean

                        yule

                type: string

                default: euclidean

        additionalProperties: False

    BaseFclusterConfig:

        The fcluster implementation is backed by scipy and for a more detailed description we refer the reader to the documentation of scipy.cluster.hierarchy.fcluster` (https://docs.scipy.org/doc/scipy/reference/generated /scipy.cluster.hierarchy.fcluster.html).

        type: object

        properties:

            depth:

                exclusiveMinimum: 0

                type: integer

                default: 2

        additionalProperties: False

    LimitedHierarchicalConfig:

        type: object

        properties:

            type:

                enum:

                        limited_hierarchical

                type: string

                default: limited_hierarchical

            linkage:

                must be:

                        LinkageConfig

                default:

                    method: average

                    metric: euclidean

            fcluster:

                must be:

                        BaseFclusterConfig

                default:

                    depth: 2

        additionalProperties: False

    LimitedKmeansClustering:

        The kmeans implementation is backed by sklearn and for a more detailed description we refer the reader to the documentation of sklearn.cluster.KMeans (https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html).

        type: object

        properties:

            type:

                enum:

                        limited_kmeans

                type: string

                default: limited_kmeans

            init:

                The criterion to use in forming flat clusters.

                enum:

                        k-means++

                        random

                type: string

                default: k-means++

            n_init:

                Number of time the k-means algorithm will be run with different centroid seeds. The final results will be the best output of n_init consecutive runs in terms of inertia.

                exclusiveMinimum: 0

                type: integer

                default: 10

            max_iter:

                Maximum number of iterations of the k-means algorithm for a single run.

                exclusiveMinimum: 0

                type: integer

                default: 300

            random_state:

                Determines random number generation for centroid initialization. Use an int to make the randomness deterministic

                exclusiveMinimum: 0

                type: integer

        additionalProperties: False

    PCAConfig:

        type: object

        properties:

            threshold:

                exclusiveMinimum: 0

                maximum: 1.0

                type: number

                default: 0.95

        additionalProperties: False

    AutoScaleConfig:

        type: object

        properties:

            type:

                enum:

                        auto_scale

                type: string

                default: auto_scale

            clustering:

                must be one of:

                        LimitedHierarchicalConfig

                        LimitedKmeansClustering

                default:

                    type: limited_hierarchical

                    linkage:

                        method: average

                        metric: euclidean

                    fcluster:

                        depth: 2

            pca:

                must be:

                        PCAConfig

                default:

                    threshold: 0.95

        additionalProperties: False

    FclusterConfig:

        The fcluster implementation is backed by scipy and for a more detailed description we refer the reader to the documentation of scipy.cluster.hierarchy.fcluster` (https://docs.scipy.org/doc/scipy/reference/generated /scipy.cluster.hierarchy.fcluster.html).

        type: object

        properties:

            depth:

                exclusiveMinimum: 0

                type: integer

                default: 2

            threshold:

                must be one of:

                    type: integer

                    type: number

                default: 1.15

            criterion:

                enum:

                        inconsistent

                        distance

                        maxclust

                        monocrit

                        maxclust_monocrit

                type: string

                default: inconsistent

        additionalProperties: False

    HierarchicalConfig:

        type: object

        properties:

            type:

                enum:

                        hierarchical

                type: string

                default: hierarchical

            linkage:

                must be:

                        LinkageConfig

                default:

                    method: average

                    metric: euclidean

            fcluster:

                must be:

                        FclusterConfig

                default:

                    depth: 2

                    threshold: 1.15

                    criterion: inconsistent

        additionalProperties: False

    KmeansClustering:

        The kmeans implementation is backed by sklearn and for a more detailed description we refer the reader to the documentation of sklearn.cluster.KMeans (https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html).

        type: object

        properties:

            type:

                enum:

                        kmeans

                type: string

                default: kmeans

            init:

                The criterion to use in forming flat clusters.

                enum:

                        k-means++

                        random

                type: string

                default: k-means++

            n_init:

                Number of time the k-means algorithm will be run with different centroid seeds. The final results will be the best output of n_init consecutive runs in terms of inertia.

                exclusiveMinimum: 0

                type: integer

                default: 10

            max_iter:

                Maximum number of iterations of the k-means algorithm for a single run.

                exclusiveMinimum: 0

                type: integer

                default: 300

            random_state:

                Determines random number generation for centroid initialization. Use an int to make the randomness deterministic

                exclusiveMinimum: 0

                type: integer

            n_clusters:

                The scalar gives the maximum number of clusters to be formed.This has a defaulted value for auto_scale and is not configurable for this workflow.

                exclusiveMinimum: 0

                type: integer

                default: 8

        additionalProperties: False

    CustomScaleConfig:

        type: object

        properties:

            type:

                enum:

                        custom_scale

                type: string

                default: custom_scale

            clustering:

                must be one of:

                        HierarchicalConfig

                        KmeansClustering

                default:

                    type: hierarchical

                    linkage:

                        method: average

                        metric: euclidean

                    fcluster:

                        depth: 2

                        threshold: 1.15

                        criterion: inconsistent

            pca:

                must be:

                        PCAConfig

                default:

                    threshold: 0.95

        additionalProperties: False

Category
    observations.correlation
Source package
    semeio
